proto/src/determined/api/v1/experiment.proto

```diff
@@ -382,14 +386,16 @@ message MetricNamesResponse {
   repeated string validation_metrics = 3;
 }
 
-// To distinguish the 2 different categories of metrics.
+// To distinguish the different categories of metrics.
 enum MetricType {
   // Zero-value (not allowed).
   METRIC_TYPE_UNSPECIFIED = 0;
   // For metrics emitted during training.
   METRIC_TYPE_TRAINING = 1;
   // For metrics emitted during validation.
   METRIC_TYPE_VALIDATION = 2;
+  // For metrics emitted during any process.
+  METRIC_TYPE_GENERIC = 3;
```

Comment:
suggestion: remove it, I don't think we need this yet? when we do it'd probably look different.